app/routes/events/EventEditRoute.js

```diff
@@ -65,12 +67,19 @@ const mapStateToProps = (state, props) => {
       feedbackRequired: valueSelector(state, 'feedbackRequired'),
       useStripe: valueSelector(state, 'useStripe'),
       priceMember: valueSelector(state, 'priceMember'),
+      separateDeadlines: valueSelector(state, 'separateDeadlines'),
       registrationDeadline:
         valueSelector(state, 'startTime') &&
         moment(valueSelector(state, 'startTime')).subtract(
           valueSelector(state, 'registrationDeadlineHours'),
           'hours'
         ),
+      unregistrationDeadline:
```

Comment:
Since this now defined two places, would it be possible to only have it in one route/in utils and import it?